maths/Ugly_numbers.dart

```diff
@@ -0,0 +1,37 @@
+int maxDivide(int a, int b) {
+  var n;
+  while (a % b == 0) {
+    n = a / b;
+    a = n.toInt();
+  }
+  return a;
+}
+
+/* Function to check if a number is ugly or not */
+int isUgly(int no) {
```

Comment:
Such function should return `bool`
```suggestion
bool isUgly(int no) {
```